src/test/java/com/example/rets_api/repository/PropertyRepositoryJPATest.java

```diff
@@ -55,6 +55,7 @@ public void should_store_a_PropertyAndSchoolAndRoom() {
         propertyToSave.setSchoolList(Arrays.asList(UtilsTest.createSchoolEntity("primarySchool", "jrHighSchool")));
         propertyToSave.setRoomList(Arrays.asList(createRoomEntity(RoomType.LIVING_ROOM), createRoomEntity(RoomType.MAIN_FLOOR_BEDROOM)));
         propertyToSave.setLotData(UtilsTest.createLotDataEntity());
+        propertyToSave.setCommunity(UtilsTest.createCommunityEntity());
```

Comment:
you can add this set to UtilsTest.createPropertyEntityWithBasicFields() so all places that call this methods will be updated

Reply:
done